styles/pages/index.less

```diff
@@ -95,7 +95,6 @@
   }
 }
 
-
```

Comment:
Is this just removing a newline? In my expeirence style adjustments usually go in their own commit / PR.

Reply:
A bit nitpicking in this context imo, but I agree it's best practice in general to make formatting commits independently to code changes

Reply:
Agreed that it's nitpicking XD. Thanks for putting up with it.